notification/src/main/java/com/amazon/opendistroforelasticsearch/alerting/destination/message/CustomWebhookMessage.java

```diff
@@ -15,6 +15,7 @@
 
 package com.amazon.opendistroforelasticsearch.alerting.destination.message;
 
+import org.apache.http.client.methods.*;
```

Comment:
NIT: Could you include all imports separately instead of *

Reply:
Fixed here and in DestinationHttpClient.java